### PR TITLE
Mypy

### PR DESCRIPTION
**Pull request by teh**
###### Motivation for this change

mypy has changed its name and versioning scheme

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -32038,6 +32038,26 @@ EOF
     };
   };
 
+  typed-ast = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: can you put this in `pkgs/development/python-modules/typed-ast/default.nix

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -32038,6 +32038,26 @@ EOF
...
+    version = "1.0.1";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/t/typed-ast/${name}.zip";
+      sha256 = "10srza5ycwzdirbgd68z0zkw7i4yd3wic1qmp00274lqajh7ixdm";
+    };
+    buildInputs = [ pkgs.unzip ];
+
+    # Only works with Python 3.3 and newer;
+    disabled = !isPy3k && !(pythonOlder "3.3");
+    propagatedBuildInputs = [ python ];
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: no need for python here

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -32038,6 +32038,26 @@ EOF
...
   };
 
+  typed-ast = buildPythonPackage rec {
+    name = "typed-ast-${version}";
+    version = "1.0.1";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/t/typed-ast/${name}.zip";
+      sha256 = "10srza5ycwzdirbgd68z0zkw7i4yd3wic1qmp00274lqajh7ixdm";
+    };
+    buildInputs = [ pkgs.unzip ];
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: don't we typically use fetchzip?

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (teh) [inline pkgs/top-level/python-packages.nix]: done.

Is the plan to create a new file for every new module?

**Comment by teh** [comment]
PTAL

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: yes

Comment (teh) [inline pkgs/top-level/python-packages.nix]: Might be worth sending a mail to the mailing list! I think I missed the discussion :)

**Pull request merged**